src/strings/normalize.c

```diff
@@ -134,6 +125,21 @@ MVMString * MVM_unicode_codepoints_to_nfg_string(MVMThreadContext *tc, const MVM
     return str;
 }
 
+/* Takes an object, which must be of VMArray representation and holding
+ * 32-bit integers. Treats them as Unicode codepoints, normalizes them at
+ * Grapheme level, and returns the resulting NFG string. */
+MVMString * MVM_unicode_codepoints_to_nfg_string(MVMThreadContext *tc, const MVMObject *codes) {
+    MVMCodepoint  *input;
+    MVMint64       input_codes;
+
+    /* Get input array; if it's empty, we're done already. */
```

Comment:
The `if it's empty` bit no longer applies; can just kill off this comment though as it doesn't add much. :-)